src/shared/modules/itemEditor/itemEditorDuck.js

```diff
@@ -85,6 +85,24 @@ export const handleFetchDataEpic = (action$, store) =>
       })
   })
 
+/**
+ * This function returns the cypher compatible value of the received value.
+ */
+let convertedValue = ''
```

Comment:
get this variable inside getCypherCompatibleValue() method

Reply:
done